components/docs-chef-io/content/automate/chef_infra_in_chef_automate.md

```diff
@@ -48,9 +54,13 @@ The list of parameters are as follows:
 | opscode_erchef['reindex_item_retries']          | 3                                         | 3                                   | ```[erchef.v1.sys.index]```<br>```reindex_item_retries=3```      |
 | opscode_erchef['cbv_cache_enabled']             | FALSE                                     | FALSE                               | ```[erchef.v1.sys.api]```<br>```cbv_cache_enabled=false```           |
 | opscode_erchef['search_queue_mode']             | batch                                     | batch                               | ```[erchef.v1.sys.index]```<br>```search_queue_mode="batch"```         |
-| opscode_erchef['s3_enabled']             | FALSE                                     | FALSE                               | ```[erchef.v1.sys.api]```<br>```s3_enabled="true"```         |
-| opscode_erchef['s3_bucket_name']             | Bookshelf                                     | Bookshelf                               | ```[erchef.v1.sys.api]```<br>```s3_bucket_name="name"```         |
-| opscode_erchef['s3_external_url']             |                                      |                                | ```[erchef.v1.sys.api]```<br>```s3_external_url="url"```         |
+| opscode_erchef['s3_enabled']             | FALSE                                     | FALSE                                      | ```[erchef.v1.sys.api]```<br>```s3_enabled="true"```         |
+| opscode_erchef['s3_bucket_name']             | Bookshelf                                     | Bookshelf                          | ```[erchef.v1.sys.api]```<br>```s3_bucket_name="name"```         |
+| opscode_erchef['s3_external_url']             |                                      |                                            | ```[erchef.v1.sys.api]```<br>```s3_external_url="url"```         |
+| opscode_erchef['strict_search_result_acls']     |FALSE                               | FALSE                                      |``` [erchef.V1.sys.api]```<br>```strict_search_result_acls= false```|
+|opscode_erchef['enable_ibrowse_traces']          |FALSE                                      |FALSE                                |```[erchef.v1.sys.ibrowse]```<br> ```enable_ibrowse_traces = false``` |
+|license['nodes']                                 |                                           |                                     |```[erchef.v1.sys.authz]```<br>  ```node_license =```              |
+|opscode_erchef['s3_url_expiry_window_size']      | 100 Percent                               |                                     |```[erchef.v1.sys.api]``` <br> ```s3_url_expiry_window_size_percent=55```|
```

Comment:
this should be 100, not 100 percent?

Reply:
Hey @YashviJain01 , The rendered sys.config reflects { 100, percent } as the parameter. But the User Have to patch config s3_url_expiry_window_size_percent=100 in order to reflect the rendered value.

Reply:
Yes, make it too 100 then. Percent is something already there in config. We have followed this while creating this doc.

Reply:
Yeah, Let me fix this one.